### ISSUE TITLE
Compatible with OS X El Capitan?

### ISSUE THREAD

**Issue by Fofer** [jcavar/SpotlightFinder#1]
This worked fine for me in OS X 10.10 (and I really appreciated it) but hasn't worked at all in OS X 10.11.  Is there anything else I can try?  Thanks...

**Comment by Fofer** [comment]
I've got it working now, but the placement of the magnifying glass is wrong?

<img width="703" alt="screen shot 2016-06-12 at 7 52 51 am" src="https://cloud.githubusercontent.com/assets/518749/15991815/ecbc821c-3072-11e6-9a60-0717bf3f7877.png">

**Comment by jcavar** [comment]
Yeah, the way this works makes it fragile when Spotlight is updated. This makes assumptions on view hierarchy and inserts button in specific view. I guess internal view hierarchy was changed in Spotlight and that broke things.

I am not sure when I will have time to investigate this, but thanks for reporting!
